zebrad/src/components/mempool.rs

```diff
@@ -562,14 +587,17 @@ impl Service<Request> for Mempool {
                 // We can't return an error since that will cause a disconnection
                 // by the peer connection handler. Therefore, return successful
                 // empty responses.
+
                 let resp = match req {
                     // Return empty responses for queries.
                     Request::TransactionIds => Response::TransactionIds(Default::default()),
 
                     Request::TransactionsById(_) => Response::Transactions(Default::default()),
                     Request::TransactionsByMinedId(_) => Response::Transactions(Default::default()),
                     #[cfg(feature = "getblocktemplate-rpcs")]
-                    Request::FullTransactions => Response::FullTransactions(Default::default()),
+                    Request::FullTransactions => {
+                        return async move { Err("no last seen chain tip hash".into()) }.boxed()
```

Comment:
Optional: do we display this error to users? (Or even developers!)

```suggestion
                        return async move { Err("mempool is not active: wait for Zebra to sync to the tip".into()) }.boxed()
```

Reply:
Applied in [`9d7c53f`](https://github.com/ZcashFoundation/zebra/pull/6416/commits/9d7c53f55dca94d66ba632ac54d833c6405823de).